src/js/eventListener.js

```diff
@@ -0,0 +1,36 @@
+import CallApi from './api.js';
+import ui from './ui.js';
+
+const addScoreForm = document.querySelector('#add-score-form');
+const refereshButton = document.getElementById('referesh-button');
+
+const submitScore = (ev) => {
+  ev.preventDefault();
+  const api = new CallApi();
+  const formInputs = Object.fromEntries(new FormData(addScoreForm).entries());
+  api.addScore(formInputs).then((data) => {
+    alert(data.result);
+  })
+    .catch((err) => console.log(err))
+    .finally(() => {
+      addScoreForm.reset();
+    });
+};
```

Comment:
- [ ] I would like us to look at this section of the project's requirements 👇🏼:

  > - JavaScript requirements
  >   - Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
  
  In the list of best practices, the use of `alert` and `console.log` is discouraged. Look into the above link and see the reasons why.
  
  Instead of `alert`, you'd rather create a pop-up. If that would be too much, you may choose the route of the suggestion below:
  
  Consider refactoring this function as below:
  
  ```JavaScript
  const submitScore = (ev) => {
    ev.preventDefault();
    const api = new CallApi();
    const formInputs = Object.fromEntries(new FormData(addScoreForm).entries());
    api.addScore(formInputs).then((data) => data)
      .catch((err) => err)
      .finally(() => {
        addScoreForm.reset();
      });
  };
  ```

Reply:
Hi @Lameck1 Thanks for the feedback, I actually thought as the next task is to implement the better styling, so will do the pop-up then. Though no excuses will follow your suggestion in the coming days too.